caller-links.json

```diff
@@ -0,0 +1,171 @@
+[
+{"zip": "35747",  "name": "Jane Ewing", "email": "contradance@charter.net" },
+{"zip": "85022",  "name": "Michael Barraclough", "email": "michael@michaelbarraclough.com", "website": "michaelbarraclough.com"},
```

Comment:
website links should start with `http://` or `https://` -- should I just assume `http://`?

Reply:
I'll go with assuming `http://`